src/preprocess.js

```diff
@@ -46,20 +46,19 @@ let preproc = q => {
 
   if (q.xxpath == "raw") {
     if (q.xxparam == "inp") return { key: "input" }
-    if (q.xxparam == "csv") {
-      // Only process the first argument which is the filename
-      // We want to extract the type info from xxparam[1] instead of evaluating it as a Rhyme query
-      let e1 = preproc(q.xxextra[0])
-      if (e1.key != "const" || typeof e1.op != "string") {
-        console.error("const string expected for filename")
-      }
-      if (q.xxextra[1] === undefined) {
-        console.error("csv schema expected")
-      }
-      return { key: "input", op: "csv", file: e1.op, schema: q.xxextra[1]}
-    }
     else if (!Number.isNaN(Number(q.xxparam))) return { key: "const", op: Number(q.xxparam) }
     else return { key: "const", op: q.xxparam }
+  } else if (q.xxpath == "loadCSV") {
+    // Only process the first argument which is the filename
+    // We want to extract the type info from xxparam[1] instead of evaluating it as a Rhyme query
+    let e1 = preproc(q.xxparam)
+    if (e1.key != "const" || typeof e1.op != "string") {
+      console.error("const string expected for filename")
+    }
```

Comment:
is this check still needed?

Reply:
I think this can be removed. I will remove it and commit to main directly.